man/systemd.exec.xml

```diff
@@ -1655,6 +1658,18 @@
       </varlistentry>
 
       <varlistentry>
+        <term><varname>RuntimeDirectoryPreserve=</varname></term>
+
+        <listitem><para>Takes one of <option>never</option>, <option>restart</option>, or <option>always</option>.
+        If set to <option>no</option> (the default), the directories specified in <varname>RuntimeDirectory=</varname>
```

Comment:
this needs to be fixed here though, as you do suggest using "no" in the current version